tfe/resource_tfe_policy_set_test.go

```diff
@@ -222,6 +224,80 @@ func TestAccTFEPolicySetImport(t *testing.T) {
 	})
 }
 
+func TestAccTFEPolicySet_vcs(t *testing.T) {
+	policySet := &tfe.PolicySet{}
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+			if v := os.Getenv("TFE_VCS_IDENTIFIER"); v == "" {
+				t.Skip("please set TFE_VCS_IDENTIFIER to run this test")
+			}
+
+			if v := os.Getenv("TFE_POLICY_SET_VCS_BRANCH"); v == "" {
+				t.Skip("please set TFE_POLICY_SET_VCS_BRANCH to run this test")
+			}
+
+			if v := os.Getenv("TFE_POLICY_SET_VCS_PATH"); v == "" {
+				t.Skip("please set TFE_POLICY_SET_VCS_PATH to run this test")
+			}
```

Comment:
I think it would be cleaner to use the same approach we used for the `GITHUB_TOKEN` env var. We define it in `provider_test.go` so that all these env vars are described at one location. And then the tests here can be updated to test against the Go vars.

Reply:
I would rather that this test be skipped on `make testacc` versus failed which is what the general pre-check function does. While (I'm assuming) GITHUB_TOKEN is generally needed to run all of the tests, someone may want to run the test suite without a VCS repository set up.

Note that this is also the first test that requires VCS access - we don't seem to be doing any workspace VCS tests. So adding these checks to fail would break any existing tests until the appropriate (live) repository fixture was added.